environment.yml

```diff
@@ -47,7 +47,9 @@ dependencies:
   - ipyevents
   - ipywidgets
   - ipympl
-  - ipyvtklink
+  - trame
```

Comment:
FYI we have to keep the `pyvista!=0.42.0` pin above because it's broken, but 0.42.1 (which should be released in the next day or so) should work fine.